ci/install_nest.sh

```diff
@@ -2,33 +2,20 @@
 
 set -e  # stop execution in case of errors
 
-if [[ "$TRAVIS_PYTHON_VERSION" == "2.7_with_system_site_packages" ]]; then
```

Comment:
Please keep this if statement; we only run the full test suite for this Python version, for the other versions NEST is not required, so we can save time by not installing it.